docs/root/start/docker.rst

```diff
@@ -3,6 +3,9 @@
 Using the Envoy Docker Image
 ============================
 
+Envoy OCI images are built using Docker and have been extensively battle-tested in large scale 
+deployments running with Docker. Use of other container technologies such as Podman, while known to
+function correctly, is not expressly supported.
```

Comment:
do we know it functions correctly? i havent tested this myself and have seen a couple of related issues

Reply:
Thanks for the info. How about language along the lines of 

"The Envoy project does not expressly support the use of container technologies other than Docker, such as Podman, even though such use might appear to function correctly in your use case."

Reply:
> Use of other container technologies such as Podman may work but have not been extensively testsed and are not expressly supported.